### ISSUE TITLE
Add packages secrets to LUCI

### ISSUE THREAD

**Issue by stuartmorgan** [flutter/flutter#131071]
In order to convert the flutter/packages FTL tests from Cirrus to LUCI, we need LUCI access to the secrets used by those tests. It looks like the FTL key is already there, but we need the Google Maps API key as well

See b/267472828 for more details about the secrets from the last time we moved them. @yusuf-goog Could you handle this? I tried to follow the instructions you pointed me to, but got a permission error on the encryption step, so it appears I don't have the necessary privileges. If the referenced API key could be uploaded as `google_maps_api_key.encrypted` that would be great.

**Comment by yusuf-goog** [comment]
Sure, let me take a look at this

**Comment by godofredoc** [comment]
@yusuf-goog any updates on this bug? any blockers that I can help with?

**Comment by yusufm** [comment]
This is not a current blocker for the port. But is expected to be something that is needed.

**Comment by yusuf-goog** [comment]
This is not a current blocker for the port. But is expected to be something that is needed.

**Comment by keyonghan** [comment]
With everything in LUCI now, do we still need the key support?

**Comment by stuartmorgan** [comment]
> With everything in LUCI now, do we still need the key support?

Yes. Running tests with the Maps API key was functionality we lost in the migration.

**Comment by keyonghan** [comment]
Ah, I see. @yusuf-goog is this still on your plate?

**Comment by matanlurey** [comment]
Adding back to ecosystem triage, the ecosystem team will need to own this if they want it over the line.

(We don't have a staffed team to implement this for you, unfortuantely)
